code/game/gamemodes/wizard/wizloadouts.dm

```diff
@@ -120,3 +120,38 @@
 	if(!user)
 		return
 	ADD_TRAIT(user, SM_HALLUCINATION_IMMUNE, MAGIC_TRAIT)
+
+/datum/spellbook_entry/loadout/fireball
+	name = "Fireball. Fireball. Fireball."
+	desc = "Who cares about the rest of the spells. Become an expert in fire magic. Devote yourself to the craft. The only spell you need anyways is <b>Fireball.</b><br>\
+		</i>Provides fire immunity, homing fireballs, rapid-fire fireballs, and some fireball wands. Provides no mobility spells. Replaces your robes with infernal versions.<i>"
+	spells_path = list(/obj/effect/proc_holder/spell/sacred_flame, /obj/effect/proc_holder/spell/fireball/homing, /obj/effect/proc_holder/spell/infinite_guns/fireball)
+	category = "Unique"
+	destroy_spellbook = TRUE
+
+/datum/spellbook_entry/loadout/fireball/OnBuy(mob/living/carbon/human/user, obj/item/spellbook/book)
+	if(!user)
+		return
+	if(user.wear_suit)
+		qdel(user.wear_suit)
+	if(user.head)
+		qdel(user.head)
+
+	to_chat(user, "<span class='notice'>You feel fireproof.</span>")
+	ADD_TRAIT(user, TRAIT_RESISTHEAT, MAGIC_TRAIT)
+	ADD_TRAIT(user, TRAIT_RESISTHIGHPRESSURE, MAGIC_TRAIT)
```

Comment:
Could you add a comment mentioning this should be refactored out at some point?

Reply:
What do you mean refactored out?

Reply:
LearnSpell isn't called when gaining a spell through something like a spell bundled , this is snowflaked in a lot of situations and will be refactored out at some point because it's gonna cause SOMEONE agony

Reply:
Oh well, they'll get 2 traits. I've added a comment to sacred flame too